catalog/pipeline/deploy-release/0.7/README.md

```diff
@@ -12,10 +12,15 @@ Tekton pipeline to verify Snapshot prior to Deployment
 | releasestrategy | The namespaced name (namespace/name) of the releaseStrategy | No | - |
 | snapshot | The namespaced name (namespace/name) of the snapshot | No | - |
 | enterpriseContractPolicy | JSON representation of the policy to be applied when validating the enterprise contract | No | - |
-| verify_ec_task_bundle | The location of the bundle containing the verify-enterprise-contract task | No | - |
+| verify_ec_task_git_url | The git repo url of the verify-enterprise-contract task | No | - |
```

Comment:
I know the previous pipeline versions all had the EC params with '-' in them...but they are not inline with ours which use camelCase. Should we use this opportunity to change it?

Reply:
We pull them straight from here https://github.com/redhat-appstudio/infra-deployments/blob/main/components/enterprise-contract/kustomization.yaml#L13
We could manipulate them, but I don't think we want to